### ISSUE TITLE
[Quest] [Tirisfal Glades] Ever So Lonely

### ISSUE THREAD

**Issue by dejmbastos** [Valhalla-Gaming/Tracker#5597]
Quest information link: http://www.wowhead.com/quest=24974/ever-so-lonely

This is the undead quest where you are supposed to catch a Murloc near the Scarlet Holdout with a tower. You're supposed to weaken him first, and then use a rope to catch him. Afterwards you need to take him back to the quest giver.

The problem with this quest, is that the murloc does not follow you, he stays exacly in the same place he was captured. If you try to walk, the rope will be attatched between you and the murloc, but after some distance the rope will break, but the murloc will still be stopped at the same spot until it disappears.

It's my first post here, so I'm sorry if I wrote too much on this.

TLDR: The murloc does not follow you.

**Comment by malex1024** [comment]
Thanks for the report. I'm working on a fix now. :)

**Comment by malex1024** [comment]
I'm locking the conversation, as the fix for the quest has been written. It'll be implemented soon.
